### ISSUE TITLE
Poster doesn't work in YouTube

### ISSUE THREAD

**Issue by terrill** [ableplayer/ableplayer#139]
There are two issues with the poster image when playing YouTube videos in Able Player:

1. It's currently dependent on the user supplying a poster attribute on the `<video>` element. In the absence of a poster attribute, it should display one of the three still images that YouTube has captured. I'm pretty sure these images are available through the YouTube API, but I'm not sure if we can determine which of the three images is the default. 

2. Even if the user has provided a poster attribute, the hack that's required in order to get caption data from the YouTube API (playing the video briefly, then restarting, as documented in my blog post [Handling Captions via the YouTube Player API](http://terrillthompson.com/blog/648)) results in the poster appearing only briefly, then disappearing in favor of a frame from early in the video (often a solid black screen).

**Comment by lukemunn** [comment]
This thread looks like it could be really helpful in terms of getting the still image paths from YouTube? 
http://stackoverflow.com/questions/2068344/how-do-i-get-a-youtube-video-thumbnail-from-the-youtube-api

**Comment by vierkantemeter** [comment]
I tried to just set the 'poster' attribute for the video element with the url for the image as described in above thread. I see it's added to the code, but it's set to 'display:none' and not visible. If I remove that in the developer console the still is visible but it's displayed on top of the video when it's playing. Does anyone know what could be done to make it work?

**Comment by terrill** [comment]
Fixed with ef569d56afdb4c2faeb90fc7775b72e0f9c3b4d9
@lukemunn the poster image isn't available directly from the YouTube iframe API. It's possible to retrieve a "full-sized" image by constructing a URL with the YouTube video ID, but those images are often smaller than the video, and in some cases they're thumbnails (even at 0.jpg). After quite a bit of testing I found that results are better if I just rely on YouTube to deliver a somewhat random still frame. Anyway, that's working again. And @vierkantemeter the poster attribute on the `<video>` element is working again too. If you add an image URL to the poster attribute that will be used in place of YouTube's default image, and it's only shown when the video is stopped with currentTime = 0
